larky/src/main/java/com/verygood/security/larky/modules/NetworkTokenModule.java

```diff
@@ -101,16 +110,32 @@ public Dict<String, Object> getNetworkToken(
       String currencyCode,
       String cryptogramType,
       String vgsMerchantId,
+      @Nullable Object transactionType,
```

Comment:
Why not just make it `String`?

Reply:
Because it's possible the passed in parameter is a `None` in larky, i.e:

`net.starlark.java.eval.NoneType`

And if it's not the case, it would be a `String`.

I checked other larky methods with potential None value, and it appears they all use `Object` type. Take vault module's `storage` parameter for example, it's also an object:

https://github.com/verygoodsecurity/starlarky/blob/81e44ad1b1b528b5557e4d1456d24f5fcd0b92a5/larky/src/main/java/com/verygood/security/larky/modules/VaultModule.java#L142

Reply:
Bah, wish they'd come up with something else like `Optional` this is annoying